### PR TITLE
updated your guidelines for core node "inject"

### PR DESCRIPTION
**Pull request by robskaar**
Hi, i'm developing an "overlay" for node-red that enables a webapp to show dashboards for interacting with flows and edit a flow in a react webapp project.
The inject core node guide line don't mention a lot about the API for the inject node, so i thought i'd supply it.
i really needed this information at this specific spot, and i had to look long to find it elsewhere, hope this helps out others who also wonder about the inject node  API.

### CODE REVIEW

File: docs/user-guide/nodes.md [hunk 1]
```diff
@@ -36,8 +36,12 @@ The `payload` can be set to a variety of different types:
  - a String, number, boolean, Buffer or Object
  - a Timestamp in milliseconds since January 1st, 1970
 
-*Since Node-RED 1.1.0*, the Inject node can now set any property on the message.
+The `interval` can maximum be set to every 596 hours (equivalent of about 24 days).
```
Comment (Steve-Mcl) [inline docs/user-guide/nodes.md:39]:
```suggestion
The `interval` can be set up to a maximum of 596 hours (approximately 24 days).
```

File: docs/user-guide/nodes.md [hunk 2]
```diff
@@ -36,8 +36,12 @@ The `payload` can be set to a variety of different types:
...
+If you are looking at intervals greater than one day - consider using a scheduler node that can cope with power outages and restarts.
+
+The `interval between tmes` and `at a specific time` options use the standard cron system. This means that 20 minutes will be at the next hour, 20 minutes past and 40 minutes past - not in 20 minutes time. If you want every 20 minutes from now - use the `interval` option.
```
Comment (Steve-Mcl) [inline docs/user-guide/nodes.md:42]:
```suggestion
The `interval between times` and `at a specific time` options use the standard cron system. This means that 20 minutes will be at the next hour, 20 minutes past and 40 minutes past - not in 20 minutes time. If you want every 20 minutes from now - use the `interval` option.
```

typo 'tmes'

**Review by Steve-Mcl — COMMENTED** [review]
Other than the 2 very small issues - looks ok to me.

**Review by Steve-Mcl — APPROVED** [review]

**Comment by Steve-Mcl** [comment]
@knolleary ok to merge please?

**Comment by knolleary** [comment]
Thanks @robskaar !

**Pull request merged**